src/styles/LoginPage.module.scss

```diff
@@ -1,81 +1,69 @@
 @import "colors";
+@import "_variables";
+@import "breakpoints";
+@import "mixins";
 
 @mixin btnStyle {
-  padding: 0.5rem 2rem;
+  padding: $padding-sm-lg;
```

Comment:
noooooo :D 2 things here. 
we won't couple two values in one. because then you would have to create n! (don't quote me on this) number of variables. let's create `spacing-sm` and `spacing-lg`, and use them both here. 
Second, naming in hard. but when saying padding in your variable name, is it ok to use it for margins? :D that is why I said that variable name is "spacing". so lets change that also